.lintstagedrc.json

```
@@ -0,0 +1,6 @@
+{
+	"$schema": "https://json.schemastore.org/lintstagedrc.schema.json",
+	"*": "prettier --ignore-unknown --write",
+	"{src/**,__tests__/**}.{mjs,js,cjs,ts,tsx}": "eslint --fix",
```

```suggestion
	"**/*.{mjs,js,cjs,ts,tsx,astro}": "eslint --fix",
```
I don't like the `__tests__` dir and besides we want to lint stuff in the root dir (like config js) anyways. also, +astro.